app/src/main/java/com/example/android/codelabs/paging/data/GithubRepository.kt

```diff
@@ -63,16 +65,33 @@ class GithubRepository(
 
         isRequestInProgress = true
```

Comment:
One feature of paging is that it can run prepend and append loads in parallel, not sure if returning when isRequestInProgress == true would prevent that?

Reply:
this will be removed when we're migrating to paging